src/main/java/org/jboss/forge/arquillian/container/provider/ContainerDirectoryParser.java

```diff
@@ -35,15 +40,23 @@ void createContainers() {
                 new TypeReference<List<Container>>() {
                 });
 
-            containers = Collections.unmodifiableList(parsedContainers);
+            Loader loader = new Loader();
+            final org.arquillian.container.chameleon.spi.model.Container[] containers = loader.loadContainers(Target.class.getClassLoader().getResourceAsStream("chameleon/default/containers.yaml"));
+
+            parsedContainers.stream()
+                .filter(container -> container.getGroupId() == null && container.getArtifactId() == null)
+                .forEach(container -> {
+                    try {
+                        container.setGroupIdAndArtifactIdFromChameleonConfiguration(containers);
```

Comment:
My main question here is - does this really should belong to `Container` object as it's coupling it with representation from Chameleon.